pages/bare_metal_cloud/dedicated_servers/bring-your-own-linux/guide.en-us.md

```diff
@@ -0,0 +1,160 @@
+---
```

Comment:
Please use guide.en-gb (change URL endpoints accordingly if needed) as source (documentation team has a script to generate all the other en pages from that)